src/components/IconDropdown/IconDropdown.jsx

```diff
@@ -156,6 +160,9 @@ const IconDropdown = ({
       ? validationText.clientHeight + defaultHelperPadding
       : 0;
 
+  const labelHeight =
+    dropdownLabel !== undefined ? dropdownLabel.clientHeight : 0;
```

Comment:
I think these measurements should be made [via a `ref` in a `useCallback`](https://reactjs.org/docs/hooks-faq.html#how-can-i-measure-a-dom-node)?

Reply:
updated, this is a better way to encapsulate this, thanks!